File: examples/java-starter-test/src/test/java/dev/choo/webscraper/UserTest.java
/*
 * Copyright (C) 2025 Steven Choo <info@stevenchoo.nl>
 *
 * MIT License. See the LICENSE file in the root directory or <http://www.opensource.org/licenses/mit-license.php>
 */

package dev.choo.webscraper;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class UserTest {

    @Test
    void canCreateUser() {
        var user = User.builder().id("id").name("name").email("email").build();
        assertThat(user.getId()).isEqualTo("id");
        assertThat(user.getName()).isEqualTo("name");
        assertThat(user.getEmail()).isEqualTo("email");
    }
}

File: examples/kotlin-starter-test/src/test/kotlin/dev/choo/webscraper/UserTest.kt
/*
 * Copyright (C) 2025 Steven Choo <info@stevenchoo.nl>
 *
 * MIT License. See the LICENSE file in the root directory or <http://www.opensource.org/licenses/mit-license.php>
 */

package dev.choo.webscraper

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class UserTest {
    @Test
    fun canCreateUser() {
        val user = User(id = "id", name = "name", email = "email")
        assertThat(user.id).isEqualTo("id")
        assertThat(user.name).isEqualTo("name")
        assertThat(user.email).isEqualTo("email")
    }
}
